blogs/lokiless_netobserv/index.md

```diff
@@ -0,0 +1,126 @@
+# Network Observability without Loki
+
+By: Mehul Modi, Steven Lee
+
+Recently, Network Observability operator 1.6 released a major enhancement to provide network insights for your OpenShift cluster without Loki. This enhancement was also featured in [What's new in Network Observability 1.6](../whats_new_1.6) blog providing quick overview of the feature. In this blog, lets look at some of the advantages and trade-offs users would have when deploying network observability with Loki disabled. As more metrics are enabled by default with this feature, we'll also demonstrate a use-case on how those metrics can benefit users for real world scenarios.
+
+# Configure Network Observability without Loki
+Loki as datasource is currently enabled by default. To configure Network Observability operator without Loki, when configuring Flowcollector resource simply set `.spec.loki.enable` to `false`
+
+```yaml
+loki:
+  enable: false
+```
+When configured as above, Network Observability's Prometheus metrics will continue to get scraped by OpenShift's cluster Prometheus without any additonal configuration and Network Traffic console will use Prometheus as a source for fetching the data.
+
+## Performance and Resource utilization gains
+
+### Query performance:
+Prometheus queries are blazing fast compared to Loki queries, but don't take my word for it, let's look at the data from the query performance tests: 
+
+Test bench environment:
+
+* Test: We did identical 50 queries for 3 separate time ranges to render topology view for both Loki and Prometheus. Such query requests all K8s Owners for the workload running in an OpenShift Cluster that had  network flows associated to them. Since we didn't have any applications running, it was all Infrastructure workloads generating network traffic. In Network Observability such unfiltered view will have topology rendered as below: 
+
+    ![unfiltered topology view](images/owner_screenshot.png)
+
+* Cluster config: 3 worker and 3 master nodes, AWS m5.2xlarge machines
+* LokiStack size: 1x.extra-small
```

Comment:
that's probably a mistake to use extra small here:
 ```
This size is ultimately dedicated for development and demo purposes.
```

Reply:
I'll re-run the test with 1x.small and gather metrics, with automation shouldn't take very long :)

Reply:
awesome ! thanks ! We'll see if it's better (or not !)